pkg/subnet/local.go

```diff
@@ -39,8 +39,8 @@ type SubnetDeployer struct {
 	backendStartedHere  bool
 }
 
-func NewLocalSubnetDeployer(app *app.Avalanche) *SubnetDeployer {
-	return &SubnetDeployer{
+func NewLocalSubnetDeployer(app *app.Avalanche) *Deployer {
```

Comment:
The reason the linter complained is because externally, these will be called `subnet.Deployer` which is better than `subnet.SubnetDeployer`. I think `subnet.LocalDeployer` and `subnet.FujiDeployer` are good choices.

Reply:
ok we may start using New() and LocalDeployer type then